R/internal.R

```diff
@@ -885,8 +884,7 @@ format.column <- function(x, single.row = FALSE, digits = 2) {
         } else {
             difference <- first.length - first.dot
         }
-        spaces <- paste(rep(" ", difference), collapse="")
-        x[i] <- paste(spaces, x[i], sep="")
+        x[i] <- paste0(x[i], sep = "")
```

Comment:
If you use `paste0`, you need to remove `, sep = ""`. That's the whole purpose of the function.

Reply:
oops :) sorry. Fixed now!